api/queries_pr.go

```diff
@@ -272,9 +280,11 @@ func PullRequests(client *Client, repo ghrepo.Interface, currentPRNumber int, cu
 						contexts(last: 100) {
 							nodes {
 								...on StatusContext {
+									context
 									state
 								}
 								...on CheckRun {
+									name
```

Comment:
Extra fields `name` and `context` are now requested in the `PullRequests` query that backs the `gh pr status` command, but is that necessary for this changeset? We only look up individual PRs in `gh pr checks`.

Reply:
agh, i committed that by accident.